src/azul/service/manifest_service.py

```diff
@@ -736,21 +741,17 @@ def _validate(cls, path: str):
         """
         msg = f'Invalid file name {path!r}'
         for character in path:
-            reject(unicodedata.category(character) == 'Cc'
-                   or character == u'\\',
-                   msg)
-        path = re.sub(r'[<>:"|?*]', '_', path).strip()
-        re_name = re.match(r'^(\.?[^./ ])([^/]*[^./ ])?(/(\.?[^./ ])([^/]*[^./ ])?)*$',
-                           path)
-        reject(re_name is None, msg)
-        assert re_name.string == path, re_name.string
-        commands = ['CON', 'PRN', 'AUX', 'NUL']
-        commands.extend([f'{cmd}{i}'
-                         for cmd in ['COM', 'LPT']
-                         for i in range(1, 10)])
-        for component in path.split('/'):
+            reject(character == u'\\' or unicodedata.category(character) == 'Cc', msg)
+        path = re.sub(r'[<>:"|?*]', '_', path)
+        path = re.fullmatch(r'^(\.?[^./ ])([^/]*[^./ ])?(/(\.?[^./ ])([^/]*[^./ ])?)*$',
```

Comment:
Use either `fullmatch` (x)or the start-of-string and end-of-string anchors, not both.